src/logic/playercommand.h

```diff
@@ -276,7 +275,7 @@ struct CmdMilitarySiteSetSoldierPreference : public PlayerCommand {
 struct CmdStartOrCancelExpedition : public PlayerCommand {
 	CmdStartOrCancelExpedition() : PlayerCommand() {
 	}  // For savegame loading
-	CmdStartOrCancelExpedition(uint32_t const t, PlayerNumber const p, Building& b)
+	CmdStartOrCancelExpedition(const Widelands::Time& t, PlayerNumber const p, Building& b)
```

Comment:
Since we moved Time to base, it should no longer be in the `Widelands` namespace.